libraries/AP_HAL_ChibiOS/hwdef/MicoAir743-AIO/defaults.parm

```diff
@@ -0,0 +1,11 @@
+#BATTERY MONITOR ON
```

Comment:
ALL of these should be defined in the hwdef file instead of a defaults.parm file in order to save unnecessary flash useage.

Reply:
Okay! Thanks for your suggestion! I just learned about this.

Reply:
Done! I have removed the default.parm file and defined the default parameters in hwdef.dat.